### PR TITLE
[New Rule] Conversions of some APT-29 Endgame rules

### LINKED ISSUE #701: [New Rule] Conversion of Endgame rules related to APT29
Related to https://github.com/elastic/protections-team/issues/345

Convert most (not all of) existing Endgame rules which were related to APT-29 and the Mitre 2019 eval.

### PR DESCRIPTION
**Pull request by brokensound77**
## Issues
Resolves #701 
Related to https://github.com/elastic/protections-team/issues/345

## Summary
Some conversions from existing Endgame rules related to APT-29

### CODE REVIEW

File: rules/windows/persistence_via_windows_management_instrumentation_event_subscription.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+risk_score = 21
+rule_id = "9b6813a1-daf1-457e-b0e6-0bb4e55b8a4c"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+  (process.name : "wmic.exe" or process.pe.original_file_name == "wmic.exe") and
+  process.args : "create"
```
Comment (Samirbous) [inline rules/windows/persistence_via_windows_management_instrumentation_event_subscription.toml:26]: create method can be used also to create a process and unrelated to wmi persistence creation, below example of wmic (needs to add more args for context e.g. ...  and process.args: ("CommandLineEventConsumer", "ActiveScriptEventConsumer") below example of cmdline:

C:\Windows\System32\wbem\WMIC.exe" /namespace:"\\root\subscription" PATH CommandLineEventConsumer CREATE Name="BotConsumer23", ExecutablePath="c:\Windows\System32\cmd.exe", CommandLineTemplate="c:\Windows\System32\cmd.exe"

File: rules/windows/discovery_admin_recon.toml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+  ((process.name : "net.exe" or process.pe.original_file_name == "net.exe") or
+   ((process.name : "net1.exe" or process.pe.original_file_name == "net1.exe") and
+       not process.parent.name : "net.exe")) and
+  process.args : ("group", "user") and
```
Comment (Samirbous) [inline rules/windows/discovery_admin_recon.toml:27]: process.ars: ("group", "user", "localgroup", "/domain")

**Review by Samirbous — CHANGES_REQUESTED** [review]

File: rules/windows/exfiltration_winrar_encryption.toml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+[metadata]
+creation_date = "2020/12/04"
+maturity = "production"
+updated_date = "2020/12/04"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies use of winrar.exe to create an encrypted rar file using the -hp option."
```
Comment (Samirbous) [inline rules/windows/exfiltration_winrar_encryption.toml:8]: can add this for reference (recent example of turla): 
https://www.welivesecurity.com/2020/12/02/turla-crutch-keeping-back-door-open/

image

Comment (Samirbous) [inline rules/windows/exfiltration_winrar_encryption.toml:8]: update description and rule name to reflect use of compression utility (7z and rar)

**Review by Samirbous — CHANGES_REQUESTED** [review]

File: rules/windows/discovery_remote_system_discovery_commands_windows.toml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+language = "eql"
+license = "Elastic License"
+name = "Remote System Discovery Commands"
+risk_score = 21
+rule_id = "0635c542-1b96-4335-9b47-126582d2c19a"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery"]
+type = "eql"
+
+query = '''
```
[the remark this answers is not in the document]
Comment (brokensound77) [reply rules/windows/discovery_remote_system_discovery_commands_windows.toml:19]: That would definitely help with FP. Do you already have a similar threshold rule for that?

[the remark this answers is not in the document]
Comment (Samirbous) [reply rules/windows/discovery_remote_system_discovery_commands_windows.toml:19]: https://github.com/elastic/detection-rules/blob/5864476fcb8809c60f370d242dc4ea16976fe6c7/rules/windows/defense_evasion_stop_process_service_threshold.toml

**Review by Samirbous — APPROVED** [review]

File: rules/windows/discovery_remote_system_discovery_commands_windows.toml (hunk 4 above)
[the remark this answers is not in the document]
Comment (brokensound77) [reply rules/windows/discovery_remote_system_discovery_commands_windows.toml:19]: yea that's a great idea. As discussed, lets push this through as is and refine this (and other related rules) to threshold in 7.12+ 👍

File: rules/windows/exfiltration_winrar_encryption.toml [hunk 5]
```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2020/12/04"
+maturity = "production"
+updated_date = "2020/12/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of winrar.exe or 7z to create an encrypted files. Adversaries will often compress and
```
Comment (bm11100) [inline rules/windows/exfiltration_winrar_encryption.toml:9]:
```suggestion
Identifies use of WinRar or 7z to create an encrypted files. Adversaries will often compress and
```
for consistency

File: rules/windows/exfiltration_winrar_encryption.toml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+process where event.type in ("start", "process_started") and
+  ((process.name:"rar.exe" or process.code_signature.subject_name == "win.rar GmbH" or
+      process.pe.original_file_name == "Command line RAR") and
+    process.args == "a" and process.args : ("-hp*", "-p*", "-dw", "-tb", "-ta", "/hp*", "/p*", "/dw", "/tb", "/ta"))
+
+  or
+  (process.pe.original_file_name in ("7z.exe", "7za.exe") and
+     process.args == "a" and process.args : ("-p*", "-sdel"))
+
+  /* uncomment if noisy for Backup SW related FPs */
```
Comment (bm11100) [inline rules/windows/exfiltration_winrar_encryption.toml:33]:
```suggestion
  /* uncomment if noisy for backup software related FPs */
```

File: rules/windows/persistence_via_windows_management_instrumentation_event_subscription.toml [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+author = ["Elastic"]
+description = """
+An adversary can use Windows Management Instrumentation (WMI) to install event filters, providers, consumers, and
+bindings that execute code when a defined event occurs. Adversaries may use the capabilities of WMI to subscribe to an
+event and execute arbitrary code when that event occurs, providing persistence on a system.
+"""
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Persistence via Windows Management Instrumentation Event Subscription"
```
Comment (bm11100) [inline rules/windows/persistence_via_windows_management_instrumentation_event_subscription.toml:16]:
```suggestion
name = "Persistence via WMI Event Subscription"
```
we use the acronym in other rule names

**Review by bm11100 — APPROVED** [review]
LGTM

**Pull request merged**